packages/dso-toolkit/components/Componenten/input-number/input-number.njk

```diff
@@ -6,12 +6,13 @@
   {% endif %}
   {% render '@button', {type: 'button', modifier: 'dso-tertiary', label: minusButtonLabel, icon: 'minus-square', iconOnly: 'true', disabled: minusButtonInactive} %}
   <input type="number"
-          id="{{ id }}"
-          readonly
-          tabindex="-1"
-          {% if min %} min="{{ min }}"{% endif %}
-          {% if max %} max="{{ max }}"{% endif %}
-          class="dso-input-step-counter"
-          value="{{ count }}">
+    id="{{ id }}"
+    readonly
+    tabindex="-1"
+    {% if min or min == 0 %} min="{{ min }}"{% endif %}
+    {% if max %} max="{{ max }}"{% endif %}
+    {% if not label and not labeledAlready %} aria-label="Aantal"{% endif %}
```

Comment:
Is `labeledAlready` echt nodig? Het is onderdeel van de API maar wordt wel gezet aan de hand van `label`.

Als het echt nodig is, kunnen we dit anders aanvliegen? `labeledAlready` is `not not label`, waarna het hier als `not labeledAlready` wordt ingezet.

Waarom kan je niet direct aan `not label` hangen? De lit implementatie in `input-number.template.ts` heeft immers ook geen `labeledAlready` property gekregen.

Reply:
Het smerige probleem hier is dat form-elements/group-input-number.njk zelf voorbeelden heeft van de group met en zonder label. En dat het daar dus van afhangt of het inliggende component zichzelf moet labelen.

Reply:
Opgelost door voor label null speciale afhandeling te introduceren.